### PR TITLE
Add this week's roundup

### PR DESCRIPTION
**Pull request by neilvanbeinum**

### CODE REVIEW

File: source/blog/2017-10-27-unboxed-roundup-our-links-for-w-c-23rd-october-2017.html.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+title: 'Roundup: GTA graphics, flying robots and frontend skillsets'
+date: '2017-10-27 13:30:00 UTC'
+authors:
+  - 'Neil van Beinum'
+tags: # (Delete as appropriate)
+- Culture
+
+---
+
+## GTA V - Graphics Study [Charlie](/people#charlie-egan)
```
Comment (borisdivjak) [inline source/blog/2017-10-27-unboxed-roundup-our-links-for-w-c-23rd-october-2017.html.markdown]: Should there be a dash before name?

File: source/blog/2017-10-27-unboxed-roundup-our-links-for-w-c-23rd-october-2017.html.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+http://www.adriancourreges.com/blog/2015/11/02/gta-v-graphics-study
+
+Saw this mentioned in the comments to a similar, [more recent write up](https://interplayoflight.wordpress.com/2017/10/25/how-unreal-renders-a-frame/) for unreal. This explains the surprisingly many things that an engine must do to build up an apparently simple video game scene. Lots of nice pictures explaining the process along the way.
+
+## Festo's Fantastical Flying Robots [Elena T](/people#elena-tanasoiu)
```
Comment (borisdivjak) [inline source/blog/2017-10-27-unboxed-roundup-our-links-for-w-c-23rd-october-2017.html.markdown]: Should there be a dash before name?

**Review by borisdivjak — CHANGES_REQUESTED** [review]
There are a couple of dashes (hyphens) missing

**Comment by neilvanbeinum** [comment]
Thanks @borisdivjak, updated with correct dash format.

**Review by borisdivjak — APPROVED** [review]
All good :)

**Pull request merged**